specification/hugr.md

```diff
@@ -494,24 +493,24 @@ flowchart
 #### Hierarchical Relationships and Constraints
 
 To clarify the possible hierarchical relationships, using the operation
-definitions above and also defining "*O"* to be all non-nested dataflow
+definitions above and also defining "_O"_ to be all non-nested dataflow
```

Comment:
I'd prefer for these unrelated changes to the spec to go in a seperate PR. That said, this line looks sus both before and after